packages/run-protocol/src/metrics.js

```diff
@@ -0,0 +1,40 @@
+// @ts-check
+import { makeStoredSubscription, makeSubscriptionKit } from '@agoric/notifier';
+import { E } from '@endo/eventual-send';
+
+const PATH = 'metrics';
```

Comment:
Maybe make the PATH an optional argument that defaults to undefined (no subpath off of the storageNode).

Reply:
yeah, that was the next step. I wasn't sure how general purpose to take on. thanks for supporting that it go in `@agoric/notifier`